docs/workforce-management-framework/topics/con-Security-Strategy-for-Keycloak.adoc

```diff
@@ -0,0 +1,28 @@
+[id='Security-Strategy-Keycloak-{chapter}']
+= Security Strategy for {WFM-RC-NameShort} {WFM-RC-KeycloakName}
+
+== Session Management
+{WFM-RC-KeycloakName} uses the Express middleware `Express Sessions` module for the {WFM-RC-NameShort} cloud server.
+Visit link:https://github.com/expressjs/session[express-session] to find more information about the available express
+session options.
+One of the key functions of {WFM-RC-KeycloakName} is that you can administer sessions on the
```

Comment:
"One of the key functions of {WFM-RC-KeycloakName} is that you can administer sessions on the...."

**>**

"`Session management can be performed` on the...."

Reply:
agree